enterprise/api_client/enterprise.py

```diff
@@ -0,0 +1,50 @@
+
+# -*- coding: utf-8 -*-
+"""
+Client for communicating with the Enterprise API.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.conf import settings
+from django.core.cache import cache
+
+from enterprise.api_client.lms import LmsApiClient
+from enterprise.utils import get_cache_key, traverse_pagination
+
+
+class EnterpriseApiClient(LmsApiClient):
```

Comment:
@brittneyexline @zubair-arbi I'll be needing this for building the Programs landing page because we plan on querying `/enterprise/api/v1/programs/{uuid}/` for Programs data rather than discovery's endpoint. Since I can't actually afford to wait for this PR to get merged, I'll copy and paste your implementation here and extend it for now.